business_requirement_deliverable_project/views/business_view.xml

```diff
@@ -41,7 +41,7 @@
                             type="object"
                             class="oe_highlight"
                             groups="business_requirement.group_business_requirement_manager"
-                            attrs="{'invisible': ['|', ('project_id', '=', False), ('state', '!=', 'approved')]}"/>
+                            attrs="{'invisible': ['|', ('project_id', '=', False), ('state', '!=', 'stakeholder_approval')]}"/>
```

Comment:
@astirpe correct.
I am back now and I am trying to check the project.
What is the status of this PR?
Runbot is down but I just forced a rebuild now